src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -238,6 +246,13 @@ public sealed partial class TokenCache : ITokenCacheInternal
             }
         }
 
+        private DateTimeOffset? CanculateSuggestedCacheExpiry()
+        {
+            IEnumerable<MsalAccessTokenCacheItem> tokenCacheItems = GetAllAccessTokensWithNoLocks(true).ToList();
```

Comment:
Why "ToList()" ?

Reply:
I was getting strange errors from intellisense that prompted this. The errors are gone now so this will be removed